MBBSEmu/HostProcess/GlobalRoutines/UsersOnlineGlobal.cs

```diff
@@ -0,0 +1,89 @@
+﻿using MBBSEmu.Extensions;
+using MBBSEmu.Memory;
+using MBBSEmu.Module;
+using MBBSEmu.Session;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using MBBSEmu.Session.Enums;
+
+namespace MBBSEmu.HostProcess.GlobalRoutines
+{
+    public class UsersOnlineGlobal : IGlobalRoutine
+    {
+        public bool ProcessCommand(byte[] command, ushort channelNumber, PointerDictionary<SessionBase> sessions, Dictionary<string, MbbsModule> modules)
+        {
+            var commandString = Encoding.ASCII.GetString(command).TrimEnd('\0');
+
+            if (commandString == "/#")
+            {
+                sessions[channelNumber].SendToClient("|RESET|\r\n|B||GREEN|LINE  USER-ID                    ....... OPTION SELECTED|RESET|\r\n".EncodeToANSIArray());
```

Comment:
essentially a constant, would be a little performance improvement to store this as a static somewhere so it doesn't have to be parsed into ANSI each time

Reply:
The ANSIStringExtensions helpers just help with giving the ANSI formatting a more readable form vs. "\0[39;49m" for colors -- even if it were static.